### ISSUE TITLE
Reset the project

### ISSUE THREAD

**Issue by thomas-dudoux** [NegreriePaul/GitHubInit#12]
We have to reset the project the to fit an MVC Style. We can use the annual project for more help and a better start. Any idea ?

**Comment by gurnavdeep** [comment]
Yes we will proceed like that. We have 2 weeks to do that. We'll follow the same MVC structure that we've used for the annual project so that we can do the unit testing of our code.
